lib/bolt/analytics.rb

```diff
@@ -21,7 +21,8 @@ module Analytics
       statement_count: :cd6,
       resource_mean: :cd7,
       plan_steps: :cd8,
-      return_type: :cd9
+      return_type: :cd9,
+      boltdir_type: :cd10
```

Comment:
I already set up cd10 for "inventory_version in google analytics. I've added cd11 for Boltdir Type now